docs/source/templates.rst

```diff
@@ -53,6 +53,28 @@ The first line again indicates that this template extends an existing template b
 The second and last line indicate that the enclosed lines are to be placed in the *body* block of the base template.
 The third line is the actual command that we want to add and the fourth line ensures that the code provided by the base template within the body block is still added.
 
+.. note::
+
+    This custom HPC `class` (see code below) should be stored in its own `python` file (named `hpc_setup.py`) in the main directory and not in the `project.py` file.
+    Storing it in the `project.py` file will result in the following warning, but the code should still execute properly.
+    You will also need to `import hpc_setup` in the `project.py` file.
```

Comment:
1. The word "class" is used as a term here, not as a code example.
2. Similar we can use the word "python" as a name in this context.
3. We should make clear that the suggested module name is an example, not something required by the module.
4. I would avoid the "HPC" semantics since environments are in principle generic.

```suggestion
    The custom environment class should be defined in and then imported from a separate Python module file (e.g., `environments.py`) in the project root directory and not in the `project.py` file to avoid an inconsequential warning.
    Defining it within in the `project.py` module will produce the warning shown below, but the code should still execute as expected.
```

Reply:
done